frontend/app/scripts/controllers/header.js

```diff
@@ -2,23 +2,60 @@
  * Created by Natsik on 11.07.2015.
  */
 angular.module('frontendApp')
-  .controller('HeaderCtrl', function ($location,  $scope) {
+  .controller('HeaderCtrl', ['EventService','$location','$scope','$aside', function (EventService,$location, $scope, $aside) {
```

Comment:
Please use follow order: standart angular dependencies is first, then library dependencies, then your services.